test-e2e/Cargo.toml

```diff
@@ -23,6 +23,7 @@ uniffi = "0.28.0"
 
 # Circom dependencies
 rust-witness = "0.1"
+witnesscalc-adapter = "0.1"
```

Comment:
this can be removed?

Reply:
and also `test-vectors/circom/witnesscalc`
since currently there is no test on witnesscalc in test-e2e

Reply:
We need this `test-vectors/circom/witnesscalc` for `mopro-ffi` testing.